DATA_SOURCES.md

```diff
@@ -431,7 +431,9 @@ For many European countries, data is available from [ENTSO-E](https://transparen
 - United States of America
   - Federal: [EIA](https://www.eia.gov/electricity/data.cfm#gencapacity)
   - Balancing Authorities: [EIA](https://www.eia.gov/electricity/data/eia860M/)
-  - States: [EIA](https://www.eia.gov/electricity/data/state/)
+  - States:
+    - [EIA](https://www.eia.gov/electricity/data/state/)
+    - [EIA](https://www.eia.gov/electricity/data/eia860M/)
```

Comment:
Is this needed with the above link on the federal level?

Reply:
yes that would probably help

Reply:
actually, the data set is listed on that page